sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,195 @@
+---
+id: high-contrast-document
```

Comment:
Since the `id` will be what shows up in the url, let's change this to just be "high-contrast".